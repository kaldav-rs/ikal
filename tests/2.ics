BEGIN:VCALENDAR
VERSION:2.0
PRODID:ownCloud Calendar
CALSCALE:GREGORIAN
BEGIN:VEVENT
UID:7e27d9fc69
DTSTAMP:20150717T101959Z
CREATED:20150717T101959Z
LAST-MODIFIED:20150717T101959Z
SUMMARY:Garage
DTSTART;VALUE=DATE:20150722
DTEND;VALUE=DATE:20150723
LOCATION:
DESCRIPTION:
CATEGORIES:
END:VEVENT
END:VCALENDAR
