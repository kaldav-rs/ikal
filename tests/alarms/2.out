Ok(
    Display(
        Display {
            action: Text {
                params: {},
                text: "DISPLAY",
            },
            trigger: Duration(
                Duration {
                    secs: -1800,
                    nanos: 0,
                },
            ),
            description: Text {
                params: {},
                text: "Breakfast meeting with executive\\nteam at 8:30 AM EST.",
            },
            duration: Some(
                Duration {
                    secs: 900,
                    nanos: 0,
                },
            ),
            repeat: Some(
                2,
            ),
            x_prop: {},
            iana_prop: {},
        },
    ),
)
